core-services/prow/02_config/openshift/cluster-baremetal-operator/_pluginconfig.yaml

```diff
@@ -6,9 +6,14 @@ approve:
 label:
   restricted_labels:
     openshift/cluster-baremetal-operator:
-    - allowed_users:
+    - allowed_teams:
+      - openshift-patch-managers
+      allowed_users:
       - sadasu
       label: backport-risk-assessed
+    - allowed_teams:
+      - openshift-patch-managers
+      label: cherry-pick-approved
```

Comment:
This should fallback to the global config, does it not?

Reply:
Is the fallback behavior per-label or is it per repo?

Reply:
Per label

Reply:
Ah, I thought it was per-plugin. I've removed these lines in the latest draft.